packages/stryker/src/utils/TempFolder.ts

```diff
@@ -0,0 +1,87 @@
+import * as fs from 'mz/fs';
+import * as path from 'path';
+import * as mkdirp from 'mkdirp';
+import * as log4js from 'log4js';
+import { deleteDir } from './fileUtils';
+
+const log = log4js.getLogger('TempFolder');
+
+export class TempFolder {
```

Comment:
We should add a `private constructor() { }` in order to prevent anyone from creating an instance them selves.

Reply:
Fixed